### PR TITLE
Add modern UI

### PR DESCRIPTION
**Pull request by jandrej**
- [x] Update `glvis.js` and `versions.js` once https://github.com/GLVis/glvis/pull/156 is merged

### CODE REVIEW

File: live/index.html [hunk 1]
```diff
@@ -1,583 +1,813 @@
...
-        } else {
-          console.warn("full screen not allowed");
-        }
+      var stream = new URLSearchParams(document.location.search).get("stream");
+      if (stream) {
+        console.log(`loading stream from ${stream}`);
+        this.glv.loadUrl(stream).then(() => {
+          this.glvIsInitializing = false;
+        });
+      } else {
```
Comment (tomstitt) [inline live/index.html:645]: Do we still want to support the example stream, for when people are using this locally?

For example:

```
else if (example_stream !== undefined) {
  this.glv.displayStream(example_stream);
} else {
  this.glv.loadUrl(this.examples[this.defaultExample].url).then(() => {
    this.glvIsInitializing = false;
  });
}

**Comment by tzanio** [comment]
`README` instructions need to be updated

**Comment by tzanio** [comment]
Please take a look at my changes in https://github.com/GLVis/glvis-js/blob/modern-ui/live/README.md and feel free to fix issues / improve the docs further!

**Review by tzanio — APPROVED** [review]

**Comment by tzanio** [comment]
Thank you so much @jandrej and @tomstitt !

**Review by tomstitt — APPROVED** [review]

**Comment by tomstitt** [comment]
@jandrej , could you run `make style` before merging? I didn't want to do it because it caused a _lot_ of differences and I don't want the history to show I made the changes last (that you actually made)

**Comment by tzanio** [comment]
Lets 🚢 it!

**Pull request merged**